src/Middleware/RequestThrottling/src/RequestThrottlingExtensions.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Aspnetcore.RequestThrottling;
+
+namespace Microsoft.AspNetCore.Builder
+{
+    /// <summary>
+    /// Extension methods for adding the <see cref="RequestThrottlingMiddleware"/> to an application.
+    /// </summary>
+    public static class RequestThrottlingExtensions
+    {
+        /// <summary>
+        /// Adds the <see cref="RequestThrottlingMiddleware"/> to limit the number of active requests within
```

Comment:
I think the terminology of being "within" the server doesn't really make sense here. All the requests are "within" the server, even the queued ones. They're just being held up in the queue. What we're doing is limiting the number of concurrent requests. Perhaps something like this:

```suggestion
        /// Adds the <see cref="RequestThrottlingMiddleware"/> to limit the number of concurrently-executing requests.
```